images/actions-runner/Dockerfile

```
@@ -1,11 +1,13 @@
+ARG BUILDED_IMAGE_TAG=2.319.2
 ARG BASE_TAG=2.319.1
```

名字调整下，然后再加两个环境变量到 image 中
- GH_ACTIONS_RUNNER_VERSION
- ACTIONS_RUNNER_VERSION

```suggestion
ARG ACTIONS_RUNNER_VERSION=2.319.2
ARG GH_ACTIONS_RUNNER_VERSION=2.319.1
```